### DISCUSSION TITLE
I can make some simple extensions

### DISCUSSION THREAD

**Discussion by Flappy25** [TurboWarp/extensions#630] [General]
Just reply or quote this post if you want me to try and make some

**Comment by LilyMakesThings** [comment]
Post processing

**Reply by Flappy25** [reply]
I've been trying for a while now but I just can't figure it out

**Reply by Flappy25** [reply]
I'll keep trying

**Comment by Flappy25** [comment]
> Post processing

I think the blending extension can do about the same I just don't know enough to interact with the scratch vm stage
